cmd/lakectl/cmd/local_init.go

```diff
@@ -40,6 +44,16 @@ func localInit(ctx context.Context, dir string, remote *uri.URI, force, updateIg
 		}
 	}
 
+	// when updating git, we need to verify that we are not trying to work on the git root directory
+	if updateIgnore {
+		if _, err := os.Stat(dir); !os.IsNotExist(err) {
+			gitDir, err := git.GetRepositoryPath(dir)
+			if err == nil && gitDir == dir {
```

Comment:
if `err` isn't `nil`, aren't you silencing an error?

Reply:
It was assuming no git or no git repo - I will update the code to be explicit